### DISCUSSION TITLE
MiDaS, fDOF, Image Filters and More

### DISCUSSION THREAD

**Discussion by WASasquatch** [comfyanonymous/ComfyUI#128] [Show and tell]
I am releasing my nodes currently on CivitAI for now, you can find them at: https://civitai.com/models/20793/was-node-suites-comfyui

![Screenshot_6](https://user-images.githubusercontent.com/1151589/226067356-1a892878-ab79-4091-b56f-47ed38f92a8b.png)
![Screenshot_4](https://user-images.githubusercontent.com/1151589/226067361-26b91607-fe0e-4549-8f9d-2c4ed8e591fe.png)

**Comment by GrimblyGorn** [comment]
For lack of any better locale to share this, here's a modded version of your sampler I made yesterday. Perhaps, it's of interest or use to you and others. It's a bit unfinished as I never got the Adv. version working correctly and simply commented it out in frustration. But it does have 2 flavors now and more options than before. Thanks for sharing all of yours thus far. I've got a few suggestions for more complex ones I haven't been able to craft yet, if you'd care to hear some.

[wasSamplers3.py.txt](https://github.com/comfyanonymous/ComfyUI/files/11019001/wasSamplers3.py.txt)

**Reply by WASasquatch** [reply]
So a denoise input? It should probably be a "scalar" or "integer" input? For like math nodes. I was planning on putting a bunch in but lower on my priorities cause I can't envision too much use currently 

And ya you can shoot ideas or questions if ya want

**Reply by GrimblyGorn** [reply]
These variants weren't so much about the denoise as externalizing more variables in general. This [pull](https://github.com/comfyanonymous/ComfyUI/pull/243) handles what I was after quite well though. Kind of makes our static variants unneeded, unless they were extended or running altered common samplers I guess.

I do want to look into making more universal types for his thing to use though, at some point. It doesn't handle all the input types currently, and I'm not sure how it could be made to handle new input types from custom_nodes but that would be handy to get worked out too probably.
